docs/authentication/saml/overview.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: SAML at Clerk (Beta)
+title: SAML (Beta)
```

Comment:
Could you update the H1 on line 6 as well? The page title is actually determined by the H1, not the frontmatter title

Reply:
Good catch, updated!